<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QPushButton" name="closer">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="w1_btn">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ballistics window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="w2_btn">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Calcs window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="w3_btn">
     <property name="text">
      <string>Drawing window</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
